comp/core/gui/guiimpl/gui.go

```diff
@@ -240,49 +229,57 @@ func serveAssets(w http.ResponseWriter, req *http.Request) {
 	w.Write(data)
 }
 
+func (g *gui) getAccessToken(w http.ResponseWriter, r *http.Request) {
+
+	intentToken := r.URL.Query().Get("intent")
+	// authToken is present in the query when the GUI is opened from the CLI
+	if intentToken == "" {
+		w.WriteHeader(http.StatusUnauthorized)
+		http.Error(w, "missing intentToken", 401)
+		return
+	}
+	if _, ok := g.intentTokens[intentToken]; !ok {
+		w.WriteHeader(http.StatusUnauthorized)
+		http.Error(w, "invalid intentToken", 401)
+		return
+	}
+
+	// Remove single use token from map
+	delete(g.intentTokens, intentToken)
+
+	// generate accessToken
+	accessToken := g.auth.GenerateAccessToken()
+
+	// accessToken, err := g.auth.Authenticate(intentToken)
+	// if authToken is valid, set the accessToken as a cookie and redirect the user to root page
```

Comment:
```suggestion
	// if accessToken is valid, set the accessToken as a cookie and redirect the user to root page
```

Reply:
I rephrased it a bit, `// set the accessToken as a cookie and redirect the user to root page`.